view/app-jarvis.js

```diff
@@ -58,6 +58,81 @@ app.controller('MainController', function($scope,$location,$rootScope,$http) {
 		}
 	});
 
+	$scope.getResponse = function(messageObj) {
+		const	data = `username=${USER}&message=${messageObj.message}`;
+		const index = $scope.messageStack.findIndex(obj => messageObj.fullDate === obj.fullDate && messageObj.text === obj.text);
+		$http({
+			url:`${URL}/message`,
+			method:'POST',
+			headers: {
+				'Content-Type': 'application/x-www-form-urlencoded'
+			},
+			data: data
+		}).then(resp => {
+			const res = (resp.data),
+				message = res['message'],
+				status = res['status'],
+				result = res['result'],
+				show = res['show'];
+			let resSuccess = false;
+			const resMessageObj = {
+				message: message,
+				sender: 'jarvis-bot',
+				time: String(new Date().getHours() + ':' + new Date().getMinutes()),
+				result: result,
+				show: false,
+				length: message.length
+			};
+
+			if (status && message === 'here are the current weather conditions') {
+				resMessageObj.result = JSON.parse(result);
+				resSuccess = true;
+			} else if (status && message === 'here is the meaning of the searched word') {
+				resSuccess = true;
+			} else if ((status === 'success' || status) && message === 'here are the top search results' ) {
+				resSuccess = true;
+			} else if ((status === 'success' || status) && message === 'here are the top search videos' ) {
+				$scope.videoDetails = result;
+				resSuccess = true;
+			} else if ((status === 'success' || status) && message === 'here are the searched images' ) {
+				resSuccess = true;
+			} else if ((status === 'success' || status) && message === 'Enter Reminder details : ') {
+				resSuccess = true;
+			} else if ((status === 'success' || status) && message === 'Here are your reminders : ') {
+				resSuccess = true;
+			} else if ((status === 'success' || status) && message === 'Enter Mail Details : ') {
+				resSuccess = true;
+			} else if ((status === 'success' || status) && (message === 'Information about the medicine : ' || message === 'Help on the given symptoms : ')) {
+				resSuccess = true;
+			} else if ((status === 'success' || status) && !show) {
+				resSuccess = true;
```

Comment:
I think these can be improved even more. why not keep the messages in a list and simply check `if  message in messageList` then `resSuccess=true`. We can do this for other cases as well.